stage #2/module #1 Modern Java/lesson #2. Structuring and encapsulation/content.md

```diff
@@ -121,9 +121,24 @@ Module types:
 - Automated Modules - added to the module path jars
 - Unnamed Modules - added to the classpath jars and classes
 
+## Hidden classes
+
+Hidden classes is a feature of JDK 15 that’s most interesting to framework developers.
+
+It allows them to create classes that cannot be used directly by the bytecode of other classes, they are intended for use by the framework itself that generate classes at run time and use them indirectly, via reflection.
+
+Standard APIs that define a class *ClassLoader::defineClass* and *Lookup::defineClass* are indifferent to whether the bytecodes of the class were generated dynamically (at run time) or statically (at compile time). 
+These APIs always <b>*define a visible class that will be used every time another class in the same loader hierarchy tries to link a class of that name*</b>.
```

Comment:
I suggest to add 'when' here '.. every time _when_ another class ..'.

Reply:
done